source/wp-content/themes/wporg-developer-2023/src/style/style.scss

```diff
@@ -633,7 +630,7 @@ pre {
 	color: var(--wp--custom--wporg-callout--color--text);
 	background-color: var(--wp--custom--wporg-callout--color--background);
 	border-width: 0;
-	border-radius: 2px;
+	border-radius: var(--wp--custom--button--border--radius);
```

Comment:
I think we should revert this, as it's a value copied from mu-plugins, see [comment](https://github.com/WordPress/wporg-developer/pull/447#discussion_r1495163913).